sycl/doc/extensions/experimental/sycl_ext_oneapi_annotated_ptr.asciidoc

```diff
@@ -182,7 +182,7 @@ properties. The implementation of the class defined here should preserve the
 information provided as compile-time constant properties through all uses of the
 pointer unless noted otherwise.
 
-.Not recommended Usage Example
+.Unsupported Usage Example
```

Comment:
I don't think we need examples for unsupported usage. Sufficient to say non-trivial-copyable is unsupported.